src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -354,6 +378,17 @@ private class TaskLauncherActor(
     promise.trySuccess(MatchedInstanceOps(offer.getId, Seq(InstanceOpWithSource(myselfAsLaunchSource, instanceOp))))
   }
 
+  private[this] def scheduleTaskOpTimeout(
```

Comment:
do we test this anywhere? Do you think it is worth testing?

Reply:
It wasn't tested in the original. And yes. Ideally we would test it.